android/lib/architecture-test/src/test/java/cz/quanti/rocketapp/android/lib/architecturetest/model/Module.kt

```diff
@@ -0,0 +1,76 @@
+package cz.quanti.rocketapp.android.lib.architecturetest.model
+
+import com.lemonappdev.konsist.api.architecture.Layer
+
+enum class Module(val moduleType: ModuleType, val platform: Platform) {
```

Comment:
dal jsem dohromady modul a platform, at je to citelnejsi. Plus doc z wiki a dependency jako u layers